### DISCUSSION TITLE
What is tensors axis in xyz plot?

### DISCUSSION THREAD

**Discussion by Filexor** [hako-mikan/sd-webui-supermerger#148] [Q&A] [answered]
I have noticed an axis in the xyz plot that is labelled "tensors".
But I couldn't find any explanation or use even in the code.
What does this axis do?
How should I use this axis?

**Comment by hako-mikan** [comment, marked as answer]
Sorry for the confusion. This is something I forgot to remove. 
This feature was eventually implemented as a `tensor `in `calcmode`.
Therefore, it is not used as an axis in XYZ plot.
